projects/plugins/crm/changelog/fix-crm-html-entities-for-special-characters-in-textareas

```diff
@@ -0,0 +1,3 @@
+Significance: patch
+Type: fixed
+Comment: For textarea fields, we are encoding special characters into HTML entities before store it on the DB
```

Comment:
Should this perhaps be a record that ends up in changelog instead of being an internal comment?
I'm mostly asking since it's a user-reported issue we're facing 😄

Reply:
Yes, you are right! I wrote it thinking like a summary of the PR not for the end user.